packages/web/src/components/track/LockedTrackDetailsTile.module.css

```diff
@@ -39,7 +39,6 @@
 }
 
 .trackImageWrapper {
-  width: 124px;
```

Comment:
This gets used in a lot of places. Just want to be sure it doesn't break the locked track details when we render from a lineup tile, giant track tile (track details page), chat tile. And probably needs to be verified for at least one of the other gate types, where we use this component inside of the "how to unlock" modal

Reply:
I undid this change because it's more properly handled by the flex  change in this PR: https://github.com/AudiusProject/audius-protocol/pull/6174/files#diff-52ed3a1a2b645c1e322be6e273302851b019b7ee04a80574fe9894d164b5ae60 (which I had not meant for you to review just yet hence the DO NOT REVIEW in the title ;) ) 

thanks though -- updating that one now to be more reflective of what change it is in of itself.